app/api/dds/v1/app_api.rb

```diff
@@ -76,7 +75,7 @@ class AppAPI < Grape::API
               Sneakers::CONFIG[:retry_error_exchange]
             ].concat(application_job_workers)
             .each do |expected_queue|
-              unless Sneakers::CONFIG[:connection].queue_exists? expected_queue
+              unless ApplicationJob.conn.queue_exists? expected_queue
```

Comment:
BTW, if you want ApplicationJob.conn to be public, you'll have to write some unit tests for it.